plugins/highlight-portal-history.js

```diff
@@ -17,33 +17,35 @@ function visited (data) {
   if (!history || !(history.visited || !history.captured)) {
     return;
   }
-
   if (history.captured) {
     setStyle(data, 'red', 1);
-  } else if (history.visited) {
+    return;
+  }
+  if (history.visited) {
     setStyle(data, 'yellow', 1);
   }
 }
 
 function notVisited (data) {
   var history = data.portal.options.data.history;
-  if (!history || history.visited || history.captured) {
+  if (!history) {
     return;
   }
-
   if (history.captured) {
-    setStyle(data, 'white', 0);
-  } else if (history.visited) {
+    return;
+  }
+  if (history.visited) {
     setStyle(data, 'yellow', 1);
+    return;
   }
+  setStyle(data, 'red', 1);
```

Comment:
Again, why all this code? We only want to portals with both capture/visited in false `state`.

Reply:
We want
No history -> no change
captured -> no change, shrink (radius: 4) or even hide the portal (radius: 0) (undecided yet)
visited only -> yellow & return
all others -> red

So to be able to handle all 4 situations the code should not interleave in if-else clauses.

Also I think the code is easier to understand and even a user is able to change the styles to his personal flavor.

Reply:
Ok then